unstructured/partition/text.py

```diff
@@ -157,6 +157,10 @@ def combine_paragraphs_less_than_min(
     return combined_paras
 
 
+def has_text(text: str):
```

Comment:
Could we call this something slightly more descriptive like maybe `is_not_empty_bullet`? Also needs a short docstring to clarify the purpose, and a type hint for the output.

Reply:
My reasoning for the name change is something like this is counterintuitive:
![image](https://github.com/Unstructured-IO/unstructured/assets/64741807/56ce9c44-91ce-4390-840b-b5a7dc138980)